get-started.md

```diff
@@ -0,0 +1,23 @@
+# How to get started in the Grafana Labs bug bounty program
+
+* **Grafana OSS** - we offer docker images that are synced with the `main` branch. This makes it easy to get started right away and be sure that you're looking for issues in an untouched instance. To spin up Grafana OSS in docker, run the following command:
+
+    `docker run -p 3000:3000 grafana/grafana-oss:main` and then visit http://localhost:3000 and login with `admin:admin`
+
+* **Mimir** - you can get started by following [Play with Grafana Mimir](https://grafana.com/tutorials/play-with-grafana-mimir/).
```

Comment:
Equally, [this documentation](https://grafana.com/docs/mimir/latest/get-started/) page feels more pertinent - any reason not to use it? The current page seems more geared towards end-users than security professionals?

Reply:
Agree, that page feels better!